### DISCUSSION TITLE
AWS - Architecture as Code

### DISCUSSION THREAD

**Discussion by dc-ms** [finos/architecture-as-code#57] [Ideas]
An interesting look at how AWS is looking at AaC with some parallels to CALM.

https://community.aws/content/2Ywf3HvjHo9P9AOBjil6B90y5o9/following-the-path-of-architecture-as-code

**Comment by rocketstack-matt** [comment]
Interesting, but despite using the term isn't this more IasC? Whilst I think IasC has a large overlap with the inputs required to ultimately be able to go from a fully documented architecture to an automatically scaffolded and deployed application, the AWS CDK seems to have the same issue of thinking about the implementation rather than the desired architecture.

Interested in the view of someone with far more IasC knowledge than myself @matthewgardner
